frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -54,6 +58,40 @@ export const personsLogic = kea<personsLogicType<PersonPaginatedResponse>>({
                 return match?.properties?.email || 'example@gmail.com'
             },
         ],
+        showSessions: [
+            (s) => [s.featureFlags],
+            (featureFlags: Record<string, string | boolean>): boolean => {
+                return !featureFlags[FEATURE_FLAGS.REMOVE_SESSIONS]
+            },
+        ],
+        showSessionRecordings: [
+            (s) => [s.featureFlags, s.currentTeam],
+            (featureFlags: Record<string, string | boolean>, currentTeam: TeamType): boolean => {
```

Comment:
You shouldn't add types to selector functions like this. Here and elsewhere. Kea/TypeScript should autodetect them. You may need to reload the type (just open the file for `personsLogicType`) or all files (there's some key shortcut) for changes to take effect, so it may feel while coding that you need to add them.